Jenkinsfile

```diff
@@ -88,7 +90,7 @@ pipeline {
         success {
           script {
             if (params.RELEASE_BUILD && !params.RELEASE_DRY_RUN) {
-              echo env.DOCKER_TAG // For now, just print the deploy tag in the console output -- later, communicate to release metadata service
+              createBuild(releaseName: params.RELEASE_NAME, serviceName: service_server, buildID: env.DOCKER_TAG, gitTag: env.GIT_TAG, rmsEnvironment: rmsSupportedEnvironment.staging) //staging for testing
```

Comment:
I generally don't like to comment on formatting, but I think newlines would preserve the git history nicely for posterity in cases where number of arguments > 3:
```groovy
createBuild(
  releaseName: params.RELEASE_NAME,
  serviceName: service_server,
  buildID: env.DOCKER_TAG,
  gitTag: env.GIT_TAG,
  rmsEnvironment: rmsSupportedEnvironment.staging
)
```
This formatting also makes the list of arguments easier to read.

Reply:
Updated formatting